opslevel/resource_opslevel_service.go

```diff
@@ -165,8 +150,12 @@ func reconcileTags(d *schema.ResourceData, service *opslevel.Service, client *op
 	}
 	tagInput := map[string]string{}
 	for _, tag := range tags {
-		key := strings.TrimSpace(strings.Split(tag, ":")[0])
-		value := strings.TrimSpace(strings.Split(tag, ":")[1])
+		parts := strings.Split(tag, ":")
+		if len(parts) != 2 {
+			return fmt.Errorf("invalid tag: '%s'", tag)
```

Comment:
lets fix this error message too - give them the exact reason why the tag is invalid - IE tag should have a very specific format with `:` etc - much like you did for the id error in property assignment

Reply:
Updated the error message to be more descriptive, just like when reading terraform property assignment IDs.

```
return fmt.Errorf("[%s] invalid tag, should be in format 'key:value' (only a single colon between the key and value, no spaces or special characters)", tag)
```